docs/pages/server-access/guides/azure-discovery.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Configure Teleport to Automatically Enroll Azure Virtual Machines (Preview)
+description: How to configure Teleport to automatically enroll Azure Virtual Machines.
+---
+
+The Teleport Discovery Service can connect to Azure and automatically
+discover and enroll virtual machines matching configured labels. It will then
+execute an install script on these discovered instances that will install
+Teleport, start it and join the cluster.
+
+In this scenario, Teleport Discovery Service uses an Azure invite token with a
+long time-to-live (TTL), so that new instances can be discovered and added to
+the Teleport cluster for the lifetime of the token.
```

Comment:
I think this detail is too low-level for the intro. Is there another place we can put it? As an alternative, is there a way to make this more general so users can understand at a high level how the feature works?

For example, I think we can put the detail about the TTL within one of the body sections.

Reply:
Step 1 covers the same info as part of the sample token file, so I think we can just remove this paragraph entirely. Let me know if you want to bring it back somewhere else.